IdentityAccess/src/Kreta/IdentityAccess/Infrastructure/Persistence/Doctrine/ORM/User/SymfonyMapping/Username.orm.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
+                          http://raw.github.com/doctrine/doctrine2/master/doctrine-mapping.xsd">
+
+    <embeddable name="Kreta\IdentityAccess\Domain\Model\User\Username">
+        <field name="username" column="username" type="string" length="150" nullable="false"/>
```

Comment:
You're right. We can check the username isn't less than 2 and isn't more than 20